### PR TITLE
Add some Equipment Icons

### PR DESCRIPTION
**Pull request by InfamousSky**

### CODE REVIEW

File: packs/equipment/bewitching-bloom-purple-iris.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+            "value": "<p>While dormant, this tattoo appears to be a simple flower bud, but when activated the flower swiftly blossoms, remaining that way until the next time you make your daily preparations. These blooms are colorful, elegant representations of purple iris flowers.</p>\n<p><strong>Activate</strong><span class=\"action-glyph\">2</span> envision</p>\n<p><strong>Frequency</strong> once per day</p>\n<hr />\n<p><strong>Effect</strong> Choose a willing ally you can see within 30 feet. Your ally takes on a regal bearing, exuding the presence of confident royalty. The bloom casts a 5th-level <em>@UUID[Compendium.pf2e.spells-srd.Item.Command] </em>spell (DC 28) selecting targets within range of the ally. Each target that fails the save must fall @UUID[Compendium.pf2e.conditionitems.Item.Prone] and pays homage to your ally.</p>"
```
Comment (stwlam) [inline packs/equipment/bewitching-bloom-purple-iris.json:9]: 😮

**Review by stwlam — CHANGES_REQUESTED** [review]
Needs to use core icons where appropriate, and also a better frying pan

File: packs/equipment/bewitching-bloom-purple-iris.json [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
+            "value": "<p>While dormant, this tattoo appears to be a simple flower bud, but when activated the flower swiftly blossoms, remaining that way until the next time you make your daily preparations. These blooms are colorful, elegant representations of purple iris flowers.</p>\n<p><strong>Activate</strong><span class=\"action-glyph\">2</span> envision</p>\n<p><strong>Frequency</strong> once per day</p>\n<hr />\n<p><strong>Effect</strong> Choose a willing ally you can see within 30 feet. Your ally takes on a regal bearing, exuding the presence of confident royalty. The bloom casts a 5th-level <em>@UUID[Compendium.pf2e.spells-srd.Item.Command]</em> spell (DC 28) selecting targets within range of the ally. Each target that fails the save must fall @UUID[Compendium.pf2e.conditionitems.Item.Prone] and pays homage to your ally.</p>"
```
Comment (TikaelSol) [inline packs/equipment/bewitching-bloom-purple-iris.json:9]:
```suggestion
            "value": "<p>While dormant, this tattoo appears to be a simple flower bud, but when activated the flower swiftly blossoms, remaining that way until the next time you make your daily preparations. These blooms are colorful, elegant representations of purple iris flowers.</p>\n<p><strong>Activate</strong><span class=\"action-glyph\">2</span> envision</p>\n<p><strong>Frequency</strong> once per day</p>\n<hr />\n<p><strong>Effect</strong> Choose a willing ally you can see within 30 feet. Your ally takes on a regal bearing, exuding the presence of confident royalty. The bloom casts a 5th-level @UUID[Compendium.pf2e.spells-srd.Item.Command] spell (DC 28) selecting targets within range of the ally. Each target that fails the save must fall @UUID[Compendium.pf2e.conditionitems.Item.Prone] and pays homage to your ally.</p>"
```

It's emphasized by CSS now, no need for the tags. Post player core someone can run something to remove all the existing tags to save however much space in compendiums.

File: packs/equipment/bewitching-bloom-white-poppy.json [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
+            "value": "<p>While dormant, this tattoo appears to be a simple flower bud, but when activated the flower swiftly blossoms, remaining that way until the next time you make your daily preparations. These blooms are colorful, elegant representations of white poppy flowers.</p>\n<p><strong>Activate</strong><span class=\"action-glyph\">2</span> envision</p>\n<p><strong>Frequency</strong> once per day</p>\n<hr />\n<p><strong>Effect</strong> Choose a willing ally you can see within 30 feet. A sense of peace overwhelms the ally. The tattoo casts <em>@UUID[Compendium.pf2e.spells-srd.Item.Calm]</em> or <em>@UUID[Compendium.pf2e.spells-srd.Item.Sanctuary]</em> (your choice) on the ally. The spell affects only the ally and lasts only until the end of your next turn. Your ally need not be willing if under the control of another creature. However, the controlling creature can attempt a @Check[type:will|dc:20] save, disrupting your activation on a success.</p>"
```
Comment (TikaelSol) [inline packs/equipment/bewitching-bloom-white-poppy.json:9]:
```suggestion
            "value": "<p>While dormant, this tattoo appears to be a simple flower bud, but when activated the flower swiftly blossoms, remaining that way until the next time you make your daily preparations. These blooms are colorful, elegant representations of white poppy flowers.</p>\n<p><strong>Activate</strong><span class=\"action-glyph\">2</span> envision</p>\n<p><strong>Frequency</strong> once per day</p>\n<hr />\n<p><strong>Effect</strong> Choose a willing ally you can see within 30 feet. A sense of peace overwhelms the ally. The tattoo casts @UUID[Compendium.pf2e.spells-srd.Item.Calm] or @UUID[Compendium.pf2e.spells-srd.Item.Sanctuary] (your choice) on the ally. The spell affects only the ally and lasts only until the end of your next turn. Your ally need not be willing if under the control of another creature. However, the controlling creature can attempt a @Check[type:will|dc:20] save, disrupting your activation on a success.</p>"
```

File: packs/equipment/wand-of-mental-purification-1st-level.json [hunk 4]
```diff
@@ -1,12 +1,12 @@
...
+            "value": "<p>Red feathers hang from the handle of this ivory wand. Holding it brings a sense of gentle calm.</p>\n<p><strong>Activate</strong> Cast a Spell</p>\n<p><strong>Frequency</strong> once per day, plus overcharge</p>\n<hr />\n<p><strong>Effect</strong> You cast 1st-level <em>@UUID[Compendium.pf2e.spells-srd.Item.Soothe]</em>, and can attempt to counteract one mental effect on the same target. Treat the <em>@UUID[Compendium.pf2e.spells-srd.Item.Soothe]</em> spell's level as 1 higher for this counteract check.</p>\n<hr />\n<p><strong>Craft Requirements</strong> Supply a casting of <em>@UUID[Compendium.pf2e.spells-srd.Item.Soothe] </em>of the appropriate level.</p>"
```
Comment (TikaelSol) [inline packs/equipment/wand-of-mental-purification-1st-level.json:9]:
```suggestion
            "value": "<p>Red feathers hang from the handle of this ivory wand. Holding it brings a sense of gentle calm.</p>\n<p><strong>Activate</strong> Cast a Spell</p>\n<p><strong>Frequency</strong> once per day, plus overcharge</p>\n<hr />\n<p><strong>Effect</strong> You cast 1st-level <em>@UUID[Compendium.pf2e.spells-srd.Item.Soothe]</em>, and can attempt to counteract one mental effect on the same target. Treat the @UUID[Compendium.pf2e.spells-srd.Item.Soothe] spell's level as 1 higher for this counteract check.</p>\n<hr />\n<p><strong>Craft Requirements</strong> Supply a casting of <em>soothe</em> of the appropriate level.</p>"
```

File: packs/equipment/wand-of-mental-purification-2nd-level.json [hunk 5]
```diff
@@ -1,12 +1,12 @@
...
+            "value": "<p>Red feathers hang from the handle of this ivory wand. Holding it brings a sense of gentle calm.</p>\n<p><strong>Activate</strong> Cast a Spell</p>\n<p><strong>Frequency</strong> once per day, plus overcharge</p>\n<hr />\n<p><strong>Effect</strong> You cast 2nd-level <em>@UUID[Compendium.pf2e.spells-srd.Item.Soothe]</em>, and can attempt to counteract one mental effect on the same target. Treat the <em>@UUID[Compendium.pf2e.spells-srd.Item.Soothe]</em> spell's level as 1 higher for this counteract check.</p>\n<hr />\n<p><strong>Craft Requirements</strong> Supply a casting of <em>@UUID[Compendium.pf2e.spells-srd.Item.Soothe] </em>of the appropriate level.</p>"
```
Comment (TikaelSol) [inline packs/equipment/wand-of-mental-purification-2nd-level.json:9]:
```suggestion
            "value": "<p>Red feathers hang from the handle of this ivory wand. Holding it brings a sense of gentle calm.</p>\n<p><strong>Activate</strong> Cast a Spell</p>\n<p><strong>Frequency</strong> once per day, plus overcharge</p>\n<hr />\n<p><strong>Effect</strong> You cast 2nd-level <em>@UUID[Compendium.pf2e.spells-srd.Item.Soothe]</em>, and can attempt to counteract one mental effect on the same target. Treat the @UUID[Compendium.pf2e.spells-srd.Item.Soothe] spell's level as 1 higher for this counteract check.</p>\n<hr />\n<p><strong>Craft Requirements</strong> Supply a casting of <em>soothe</em> of the appropriate level.</p>"
```

File: packs/equipment/wand-of-mental-purification-3rd-level.json [hunk 6]
```diff
@@ -1,12 +1,12 @@
...
+            "value": "<p>Red feathers hang from the handle of this ivory wand. Holding it brings a sense of gentle calm.</p>\n<p><strong>Activate</strong> Cast a Spell</p>\n<p><strong>Frequency</strong> once per day, plus overcharge</p>\n<hr />\n<p><strong>Effect</strong> You cast 3rd-level <em>@UUID[Compendium.pf2e.spells-srd.Item.Soothe]</em>, and can attempt to counteract one mental effect on the same target. Treat the <em>@UUID[Compendium.pf2e.spells-srd.Item.Soothe]</em> spell's level as 1 higher for this counteract check.</p>\n<hr />\n<p><strong>Craft Requirements</strong> Supply a casting of <em>@UUID[Compendium.pf2e.spells-srd.Item.Soothe] </em>of the appropriate level.</p>"
```
Comment (TikaelSol) [inline packs/equipment/wand-of-mental-purification-3rd-level.json:9]:
```suggestion
            "value": "<p>Red feathers hang from the handle of this ivory wand. Holding it brings a sense of gentle calm.</p>\n<p><strong>Activate</strong> Cast a Spell</p>\n<p><strong>Frequency</strong> once per day, plus overcharge</p>\n<hr />\n<p><strong>Effect</strong> You cast 3rd-level <em>@UUID[Compendium.pf2e.spells-srd.Item.Soothe]</em>, and can attempt to counteract one mental effect on the same target. Treat the @UUID[Compendium.pf2e.spells-srd.Item.Soothe] spell's level as 1 higher for this counteract check.</p>\n<hr />\n<p><strong>Craft Requirements</strong> Supply a casting of <em>soothe</em> of the appropriate level.</p>"
```

File: packs/equipment/wand-of-mental-purification-4th-level.json [hunk 7]
```diff
@@ -1,12 +1,12 @@
...
+            "value": "<p>Red feathers hang from the handle of this ivory wand. Holding it brings a sense of gentle calm.</p>\n<p><strong>Activate</strong> Cast a Spell</p>\n<p><strong>Frequency</strong> once per day, plus overcharge</p>\n<hr />\n<p><strong>Effect</strong> You cast 4th-level <em>@UUID[Compendium.pf2e.spells-srd.Item.Soothe]</em>, and can attempt to counteract one mental effect on the same target. Treat the <em>@UUID[Compendium.pf2e.spells-srd.Item.Soothe]</em> spell's level as 1 higher for this counteract check.</p>\n<hr />\n<p><strong>Craft Requirements</strong> Supply a casting of <em>@UUID[Compendium.pf2e.spells-srd.Item.Soothe] </em>of the appropriate level.</p>"
```
Comment (TikaelSol) [inline packs/equipment/wand-of-mental-purification-4th-level.json:9]:
```suggestion
            "value": "<p>Red feathers hang from the handle of this ivory wand. Holding it brings a sense of gentle calm.</p>\n<p><strong>Activate</strong> Cast a Spell</p>\n<p><strong>Frequency</strong> once per day, plus overcharge</p>\n<hr />\n<p><strong>Effect</strong> You cast 4th-level <em>@UUID[Compendium.pf2e.spells-srd.Item.Soothe]</em>, and can attempt to counteract one mental effect on the same target. Treat the @UUID[Compendium.pf2e.spells-srd.Item.Soothe] spell's level as 1 higher for this counteract check.</p>\n<hr />\n<p><strong>Craft Requirements</strong> Supply a casting of <em>soothe</em> of the appropriate level.</p>"
```

File: packs/equipment/wand-of-mental-purification-5th-level.json [hunk 8]
```diff
@@ -1,12 +1,12 @@
...
+            "value": "<p>Red feathers hang from the handle of this ivory wand. Holding it brings a sense of gentle calm.</p>\n<p><strong>Activate</strong> Cast a Spell</p>\n<p><strong>Frequency</strong> once per day, plus overcharge</p>\n<hr />\n<p><strong>Effect</strong> You cast 5th-level <em>@UUID[Compendium.pf2e.spells-srd.Item.Soothe]</em>, and can attempt to counteract one mental effect on the same target. Treat the <em>@UUID[Compendium.pf2e.spells-srd.Item.Soothe]</em> spell's level as 1 higher for this counteract check.</p>\n<hr />\n<p><strong>Craft Requirements</strong> Supply a casting of <em>@UUID[Compendium.pf2e.spells-srd.Item.Soothe] </em>of the appropriate level.</p>"
```
Comment (TikaelSol) [inline packs/equipment/wand-of-mental-purification-5th-level.json:9]:
```suggestion
            "value": "<p>Red feathers hang from the handle of this ivory wand. Holding it brings a sense of gentle calm.</p>\n<p><strong>Activate</strong> Cast a Spell</p>\n<p><strong>Frequency</strong> once per day, plus overcharge</p>\n<hr />\n<p><strong>Effect</strong> You cast 5th-level <em>@UUID[Compendium.pf2e.spells-srd.Item.Soothe]</em>, and can attempt to counteract one mental effect on the same target. Treat the @UUID[Compendium.pf2e.spells-srd.Item.Soothe] spell's level as 1 higher for this counteract check.</p>\n<hr />\n<p><strong>Craft Requirements</strong> Supply a casting of <em>soothe</em> of the appropriate level.</p>"
```

File: packs/equipment/wand-of-mental-purification-6th-level.json [hunk 9]
```diff
@@ -1,12 +1,12 @@
...
+            "value": "<p>Red feathers hang from the handle of this ivory wand. Holding it brings a sense of gentle calm.</p>\n<p><strong>Activate</strong> Cast a Spell</p>\n<p><strong>Frequency</strong> once per day, plus overcharge</p>\n<hr />\n<p><strong>Effect</strong> You cast 6th-level <em>@UUID[Compendium.pf2e.spells-srd.Item.Soothe]</em>, and can attempt to counteract one mental effect on the same target. Treat the <em>@UUID[Compendium.pf2e.spells-srd.Item.Soothe]</em> spell's level as 1 higher for this counteract check.</p>\n<hr />\n<p><strong>Craft Requirements</strong> Supply a casting of <em>@UUID[Compendium.pf2e.spells-srd.Item.Soothe] </em>of the appropriate level.</p>"
```
Comment (TikaelSol) [inline packs/equipment/wand-of-mental-purification-6th-level.json:9]:
```suggestion
            "value": "<p>Red feathers hang from the handle of this ivory wand. Holding it brings a sense of gentle calm.</p>\n<p><strong>Activate</strong> Cast a Spell</p>\n<p><strong>Frequency</strong> once per day, plus overcharge</p>\n<hr />\n<p><strong>Effect</strong> You cast 6th-level <em>@UUID[Compendium.pf2e.spells-srd.Item.Soothe]</em>, and can attempt to counteract one mental effect on the same target. Treat the @UUID[Compendium.pf2e.spells-srd.Item.Soothe] spell's level as 1 higher for this counteract check.</p>\n<hr />\n<p><strong>Craft Requirements</strong> Supply a casting of <em>soothe</em> of the appropriate level.</p>"
```

File: packs/equipment/wand-of-mental-purification-7th-level.json [hunk 10]
```diff
@@ -1,12 +1,12 @@
...
+            "value": "<p>Red feathers hang from the handle of this ivory wand. Holding it brings a sense of gentle calm.</p>\n<p><strong>Activate</strong> Cast a Spell</p>\n<p><strong>Frequency</strong> once per day, plus overcharge</p>\n<hr />\n<p><strong>Effect</strong> You cast 7th-level <em>@UUID[Compendium.pf2e.spells-srd.Item.Soothe]</em>, and can attempt to counteract one mental effect on the same target. Treat the <em>@UUID[Compendium.pf2e.spells-srd.Item.Soothe]</em> spell's level as 1 higher for this counteract check.</p>\n<hr />\n<p><strong>Craft Requirements</strong> Supply a casting of <em>@UUID[Compendium.pf2e.spells-srd.Item.Soothe] </em>of the appropriate level.</p>"
```
Comment (TikaelSol) [inline packs/equipment/wand-of-mental-purification-7th-level.json:9]:
```suggestion
            "value": "<p>Red feathers hang from the handle of this ivory wand. Holding it brings a sense of gentle calm.</p>\n<p><strong>Activate</strong> Cast a Spell</p>\n<p><strong>Frequency</strong> once per day, plus overcharge</p>\n<hr />\n<p><strong>Effect</strong> You cast 7th-level <em>@UUID[Compendium.pf2e.spells-srd.Item.Soothe]</em>, and can attempt to counteract one mental effect on the same target. Treat the @UUID[Compendium.pf2e.spells-srd.Item.Soothe] spell's level as 1 higher for this counteract check.</p>\n<hr />\n<p><strong>Craft Requirements</strong> Supply a casting of <em>soothe</em> of the appropriate level.</p>"
```

File: packs/equipment/wand-of-mental-purification-8th-level.json [hunk 11]
```diff
@@ -1,12 +1,12 @@
...
+            "value": "<p>Red feathers hang from the handle of this ivory wand. Holding it brings a sense of gentle calm.</p>\n<p><strong>Activate</strong> Cast a Spell</p>\n<p><strong>Frequency</strong> once per day, plus overcharge</p>\n<hr />\n<p><strong>Effect</strong> You cast 8th-level <em>@UUID[Compendium.pf2e.spells-srd.Item.Soothe]</em>, and can attempt to counteract one mental effect on the same target. Treat the <em>@UUID[Compendium.pf2e.spells-srd.Item.Soothe]</em> spell's level as 1 higher for this counteract check.</p>\n<hr />\n<p><strong>Craft Requirements</strong> Supply a casting of <em>@UUID[Compendium.pf2e.spells-srd.Item.Soothe] </em>of the appropriate level.</p>"
```
Comment (TikaelSol) [inline packs/equipment/wand-of-mental-purification-8th-level.json:9]:
```suggestion
            "value": "<p>Red feathers hang from the handle of this ivory wand. Holding it brings a sense of gentle calm.</p>\n<p><strong>Activate</strong> Cast a Spell</p>\n<p><strong>Frequency</strong> once per day, plus overcharge</p>\n<hr />\n<p><strong>Effect</strong> You cast 8th-level <em>@UUID[Compendium.pf2e.spells-srd.Item.Soothe]</em>, and can attempt to counteract one mental effect on the same target. Treat the @UUID[Compendium.pf2e.spells-srd.Item.Soothe] spell's level as 1 higher for this counteract check.</p>\n<hr />\n<p><strong>Craft Requirements</strong> Supply a casting of <em>soothe</em> of the appropriate level.</p>"
```

File: packs/equipment/wand-of-mental-purification-9th-level.json [hunk 12]
```diff
@@ -1,12 +1,12 @@
...
+            "value": "<p>Red feathers hang from the handle of this ivory wand. Holding it brings a sense of gentle calm.</p>\n<p><strong>Activate</strong> Cast a Spell</p>\n<p><strong>Frequency</strong> once per day, plus overcharge</p>\n<hr />\n<p><strong>Effect</strong> You cast 9th-level <em>@UUID[Compendium.pf2e.spells-srd.Item.Soothe]</em>, and can attempt to counteract one mental effect on the same target. Treat the <em>@UUID[Compendium.pf2e.spells-srd.Item.Soothe]</em> spell's level as 1 higher for this counteract check.</p>\n<hr />\n<p><strong>Craft Requirements</strong> Supply a casting of <em>@UUID[Compendium.pf2e.spells-srd.Item.Soothe] </em>of the appropriate level.</p>"
```
Comment (TikaelSol) [inline packs/equipment/wand-of-mental-purification-9th-level.json:9]:
```suggestion
            "value": "<p>Red feathers hang from the handle of this ivory wand. Holding it brings a sense of gentle calm.</p>\n<p><strong>Activate</strong> Cast a Spell</p>\n<p><strong>Frequency</strong> once per day, plus overcharge</p>\n<hr />\n<p><strong>Effect</strong> You cast 9th-level <em>@UUID[Compendium.pf2e.spells-srd.Item.Soothe]</em>, and can attempt to counteract one mental effect on the same target. Treat the @UUID[Compendium.pf2e.spells-srd.Item.Soothe] spell's level as 1 higher for this counteract check.</p>\n<hr />\n<p><strong>Craft Requirements</strong> Supply a casting of <em>soothe</em> of the appropriate level.</p>"
```

**Review by stwlam — APPROVED** [review]

**Pull request merged**